features/signup.feature

```diff
@@ -0,0 +1,192 @@
+Feature: Manage signups in a multisite installation
```

Comment:
Formatting ended up a little funky in this file

Reply:
I don't understand. Can you please elaborate?

Reply:
@ernilambar Sorry, to clarify: the formatting is incorrect.

![CleanShot 2024-04-27 at 05 50 49@2x](https://github.com/wp-cli/entity-command/assets/36432/849477ad-d37a-4301-b223-41069d7227bb)